docs/staking-and-delegation/managing-stake-btcli.md

```diff
@@ -33,11 +33,17 @@ Test network tokens have no real value. Before managing liquidity on Bittensor m
 
 ## View TAO balance
 
-To stake, you'll first need some TAO. Inquire in Discord to obtain TAO on Bittensor test network.
+To stake, you'll first need some TAO. Inquire in [Discord](https://discord.com/channels/799672011265015819/1107738550373454028/threads/1331693251589312553) to obtain TAO on Bittensor test network. Alternatively, you can transfer some testnet TAO to your wallet address using the [BTCLI Live Coding Playground](../btcli/btcli-playground.md#transfer).
 
-To ensure you are targeting the test network, run `btcli config set`, select network, and set it to `test`.
+:::info Note
+
```

Comment:
Same, I think this note should get chopped and we just have it once in teh playground page itself